src/main/java/org/folio/service/orders/OrderRolloverService.java

```diff
@@ -217,7 +217,7 @@ private Future<Void> rolloverOrdersByFundIds(List<String> chunkFundIds, LedgerFi
   private Future<Void> saveOrderLines(List<PoLine> orderLines, LedgerFiscalYearRollover ledgerFYRollover,
                                       PurchaseOrder.WorkflowStatus workflowStatus, RequestContext requestContext) {
     logger.info("saveOrderLines:: Saving POLs after rollover processing, size: {}", orderLines.size());
-    return purchaseOrderLineService.saveOrderLines(orderLines, requestContext)
+    return purchaseOrderLineService.saveOrderLinesWithoutSearchLocationsUpdate(orderLines, requestContext)
```

Comment:
performed some optimization to avoid so many requests to fetch holdings to update searchLocationId, because rollover does not update locations in Po Line, so we don't need so many new calls.
Related ticket https://folio-org.atlassian.net/browse/MODORDSTOR-419